### ISSUE TITLE
The format of the bogota_grid.tif is not correct (GEOTIFF file model from the library)

### ISSUE THREAD

**Issue by benoitgaudou** [gama-platform/gama#2789]
**Describe the bug**
When I run the model GEOTIFF File import.gaml (Data / Data Importation in the model library), I get the following error:
"The format of bogota_grid.tif is not correct." (when applying envelope (grid_data) ).

As a complement, this exception appears to me only on a new workspace. When I run GAMA on a new worskspace and run this model, I get the Exception. But if I close GAMA and run it again on the same workspace, I do not have this issue and the file is well-recognized.

The exeptions that appear in the Eclipse console are: 
```
AVERTISSEMENT: Unable to create a Reader for File: /Users/ben/Dev/GitRepository/gama/msi.gama.models/models/Data/Data Importation/includes/bogota_grid.tif
java.lang.IllegalArgumentException
	at com.sun.imageio.spi.InputStreamImageInputStreamSpi.createInputStreamInstance(InputStreamImageInputStreamSpi.java:74)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider$SpiHelper.<init>(MaskOverviewProvider.java:720)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider.<init>(MaskOverviewProvider.java:106)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider.<init>(MaskOverviewProvider.java:101)
	at org.geotools.gce.geotiff.GeoTiffReader.getHRInfo(GeoTiffReader.java:357)
	at org.geotools.gce.geotiff.GeoTiffReader.<init>(GeoTiffReader.java:249)
	at msi.gama.util.file.GamaGridFile.getOwnCRS(GamaGridFile.java:409)
	at msi.gama.util.file.GamaGisFile.getExistingCRS(GamaGisFile.java:83)
	at msi.gama.util.file.GamaGridFile$GamaGridReader.<init>(GamaGridFile.java:155)
	at msi.gama.util.file.GamaGridFile.createReader(GamaGridFile.java:87)
	at msi.gama.util.file.GamaGridFile.fillBuffer(GamaGridFile.java:340)
	at msi.gama.util.file.GamaGridFile.computeEnvelope(GamaGridFile.java:326)
	at msi.gama.common.geometry.GeometryUtils.computeEnvelopeFrom(GeometryUtils.java:832)
	at msi.gaml.operators.Spatial$Creation.envelope(Spatial.java:1407)
	at gaml.additions.GamlAdditions.lambda$356(GamlAdditions.java:413)
	at msi.gaml.expressions.UnaryOperator._value(UnaryOperator.java:83)
	at msi.gaml.expressions.AbstractExpression.value(AbstractExpression.java:70)
	at msi.gama.runtime.ExecutionScope.lambda$5(ExecutionScope.java:447)
	at msi.gama.runtime.ExecutionScope.runAndCatch(ExecutionScope.java:464)
	at msi.gama.runtime.ExecutionScope.pushRunAndCatch(ExecutionScope.java:454)
	at msi.gama.runtime.ExecutionScope.evaluate(ExecutionScope.java:447)
	at msi.gaml.variables.Variable.initializeWith(Variable.java:484)
	at msi.gama.metamodel.population.GamaPopulation.createAndUpdateVariablesFor(GamaPopulation.java:414)
	at msi.gama.metamodel.population.GamaPopulation.createVariablesFor(GamaPopulation.java:394)
	at msi.gama.kernel.simulation.SimulationPopulation.initSimulation(SimulationPopulation.java:121)
	at msi.gama.kernel.simulation.SimulationPopulation.createAgents(SimulationPopulation.java:96)
	at msi.gama.kernel.experiment.ExperimentAgent.createSimulation(ExperimentAgent.java:292)
	at msi.gama.kernel.experiment.ExperimentAgent._init_(ExperimentAgent.java:246)
	at msi.gama.metamodel.agent.MinimalAgent.init(MinimalAgent.java:216)
	at msi.gama.kernel.experiment.ExperimentAgent.init(ExperimentAgent.java:260)
	at msi.gama.runtime.ExecutionScope.lambda$3(ExecutionScope.java:440)
	at msi.gama.runtime.ExecutionScope.runAndCatch(ExecutionScope.java:464)
	at msi.gama.runtime.ExecutionScope.pushRunAndCatch(ExecutionScope.java:454)
	at msi.gama.runtime.ExecutionScope.init(ExecutionScope.java:440)
	at msi.gama.kernel.experiment.ExperimentScheduler.schedule(ExperimentScheduler.java:149)
	at msi.gama.kernel.experiment.ExperimentAgent.schedule(ExperimentAgent.java:312)
	at msi.gama.kernel.experiment.ExperimentPlan.open(ExperimentPlan.java:512)
	at msi.gama.kernel.experiment.ExperimentPlan.open(ExperimentPlan.java:522)
	at msi.gama.kernel.experiment.ExperimentController.lambda$0(ExperimentController.java:94)
	at java.lang.Thread.run(Thread.java:748)

org.geotools.data.DataSourceException: java.lang.IllegalArgumentException
	at org.geotools.gce.geotiff.GeoTiffReader.<init>(GeoTiffReader.java:262)
	at msi.gama.util.file.GamaGridFile.getOwnCRS(GamaGridFile.java:409)
	at msi.gama.util.file.GamaGisFile.getExistingCRS(GamaGisFile.java:83)
	at msi.gama.util.file.GamaGridFile$GamaGridReader.<init>(GamaGridFile.java:155)
	at msi.gama.util.file.GamaGridFile.createReader(GamaGridFile.java:87)
	at msi.gama.util.file.GamaGridFile.fillBuffer(GamaGridFile.java:340)
	at msi.gama.util.file.GamaGridFile.computeEnvelope(GamaGridFile.java:326)
	at msi.gama.common.geometry.GeometryUtils.computeEnvelopeFrom(GeometryUtils.java:832)
	at msi.gaml.operators.Spatial$Creation.envelope(Spatial.java:1407)
	at gaml.additions.GamlAdditions.lambda$356(GamlAdditions.java:413)
	at msi.gaml.expressions.UnaryOperator._value(UnaryOperator.java:83)
	at msi.gaml.expressions.AbstractExpression.value(AbstractExpression.java:70)
	at msi.gama.runtime.ExecutionScope.lambda$5(ExecutionScope.java:447)
	at msi.gama.runtime.ExecutionScope.runAndCatch(ExecutionScope.java:464)
	at msi.gama.runtime.ExecutionScope.pushRunAndCatch(ExecutionScope.java:454)
	at msi.gama.runtime.ExecutionScope.evaluate(ExecutionScope.java:447)
	at msi.gaml.variables.Variable.initializeWith(Variable.java:484)
	at msi.gama.metamodel.population.GamaPopulation.createAndUpdateVariablesFor(GamaPopulation.java:414)
	at msi.gama.metamodel.population.GamaPopulation.createVariablesFor(GamaPopulation.java:394)
	at msi.gama.kernel.simulation.SimulationPopulation.initSimulation(SimulationPopulation.java:121)
	at msi.gama.kernel.simulation.SimulationPopulation.createAgents(SimulationPopulation.java:96)
	at msi.gama.kernel.experiment.ExperimentAgent.createSimulation(ExperimentAgent.java:292)
	at msi.gama.kernel.experiment.ExperimentAgent._init_(ExperimentAgent.java:246)
	at msi.gama.metamodel.agent.MinimalAgent.init(MinimalAgent.java:216)
	at msi.gama.kernel.experiment.ExperimentAgent.init(ExperimentAgent.java:260)
	at msi.gama.runtime.ExecutionScope.lambda$3(ExecutionScope.java:440)
	at msi.gama.runtime.ExecutionScope.runAndCatch(ExecutionScope.java:464)
	at msi.gama.runtime.ExecutionScope.pushRunAndCatch(ExecutionScope.java:454)
	at msi.gama.runtime.ExecutionScope.init(ExecutionScope.java:440)
	at msi.gama.kernel.experiment.ExperimentScheduler.schedule(ExperimentScheduler.java:149)
	at msi.gama.kernel.experiment.ExperimentAgent.schedule(ExperimentAgent.java:312)
	at msi.gama.kernel.experiment.ExperimentPlan.open(ExperimentPlan.java:512)
	at msi.gama.kernel.experiment.ExperimentPlan.open(ExperimentPlan.java:522)
	at msi.gama.kernel.experiment.ExperimentController.lambda$0(ExperimentController.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.geotools.data.DataSourceException: java.lang.IllegalArgumentException
	at org.geotools.gce.geotiff.GeoTiffReader.getHRInfo(GeoTiffReader.java:487)
	at org.geotools.gce.geotiff.GeoTiffReader.<init>(GeoTiffReader.java:249)
	... 34 more
Caused by: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider$SpiHelper.<init>(MaskOverviewProvider.java:729)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider.<init>(MaskOverviewProvider.java:106)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider.<init>(MaskOverviewProvider.java:101)
	at org.geotools.gce.geotiff.GeoTiffReader.getHRInfo(GeoTiffReader.java:357)
	... 35 more
Caused by: java.lang.IllegalArgumentException
	at com.sun.imageio.spi.InputStreamImageInputStreamSpi.createInputStreamInstance(InputStreamImageInputStreamSpi.java:74)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider$SpiHelper.<init>(MaskOverviewProvider.java:720)
	... 38 more
juil. 18, 2019 10:59:26 AM org.geotools.coverage.grid.io.imageio.MaskOverviewProvider$SpiHelper <init>
AVERTISSEMENT: Unable to create a Reader for File: /Users/ben/Dev/GitRepository/gama/msi.gama.models/models/Data/Data Importation/includes/bogota_grid.tif
java.lang.IllegalArgumentException
	at com.sun.imageio.spi.InputStreamImageInputStreamSpi.createInputStreamInstance(InputStreamImageInputStreamSpi.java:74)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider$SpiHelper.<init>(MaskOverviewProvider.java:720)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider.<init>(MaskOverviewProvider.java:106)
	at org.geotools.coverage.grid.io.imageio.MaskOverviewProvider.<init>(MaskOverviewProvider.java:101)
	at org.geotools.gce.geotiff.GeoTiffReader.getHRInfo(GeoTiffReader.java:357)
	at org.geotools.gce.geotiff.GeoTiffReader.<init>(GeoTiffReader.java:249)
	at msi.gama.util.file.GamaGridFile$GamaGridReader.<init>(GamaGridFile.java:160)
	at msi.gama.util.file.GamaGridFile.createReader(GamaGridFile.java:87)
	at msi.gama.util.file.GamaGridFile.fillBuffer(GamaGridFile.java:340)
	at msi.gama.util.file.GamaGridFile.computeEnvelope(GamaGridFile.java:326)
	at msi.gama.common.geometry.GeometryUtils.computeEnvelopeFrom(GeometryUtils.java:832)
	at msi.gaml.operators.Spatial$Creation.envelope(Spatial.java:1407)
	at gaml.additions.GamlAdditions.lambda$356(GamlAdditions.java:413)
	at msi.gaml.expressions.UnaryOperator._value(UnaryOperator.java:83)
	at msi.gaml.expressions.AbstractExpression.value(AbstractExpression.java:70)
	at msi.gama.runtime.ExecutionScope.lambda$5(ExecutionScope.java:447)
	at msi.gama.runtime.ExecutionScope.runAndCatch(ExecutionScope.java:464)
	at msi.gama.runtime.ExecutionScope.pushRunAndCatch(ExecutionScope.java:454)
	at msi.gama.runtime.ExecutionScope.evaluate(ExecutionScope.java:447)
	at msi.gaml.variables.Variable.initializeWith(Variable.java:484)
	at msi.gama.metamodel.population.GamaPopulation.createAndUpdateVariablesFor(GamaPopulation.java:414)
	at msi.gama.metamodel.population.GamaPopulation.createVariablesFor(GamaPopulation.java:394)
	at msi.gama.kernel.simulation.SimulationPopulation.initSimulation(SimulationPopulation.java:121)
	at msi.gama.kernel.simulation.SimulationPopulation.createAgents(SimulationPopulation.java:96)
	at msi.gama.kernel.experiment.ExperimentAgent.createSimulation(ExperimentAgent.java:292)
	at msi.gama.kernel.experiment.ExperimentAgent._init_(ExperimentAgent.java:246)
	at msi.gama.metamodel.agent.MinimalAgent.init(MinimalAgent.java:216)
	at msi.gama.kernel.experiment.ExperimentAgent.init(ExperimentAgent.java:260)
	at msi.gama.runtime.ExecutionScope.lambda$3(ExecutionScope.java:440)
	at msi.gama.runtime.ExecutionScope.runAndCatch(ExecutionScope.java:464)
	at msi.gama.runtime.ExecutionScope.pushRunAndCatch(ExecutionScope.java:454)
	at msi.gama.runtime.ExecutionScope.init(ExecutionScope.java:440)
	at msi.gama.kernel.experiment.ExperimentScheduler.schedule(ExperimentScheduler.java:149)
	at msi.gama.kernel.experiment.ExperimentAgent.schedule(ExperimentAgent.java:312)
	at msi.gama.kernel.experiment.ExperimentPlan.open(ExperimentPlan.java:512)
	at msi.gama.kernel.experiment.ExperimentPlan.open(ExperimentPlan.java:522)
	at msi.gama.kernel.experiment.ExperimentController.lambda$0(ExperimentController.java:94)
	at java.lang.Thread.run(Thread.java:748)
```

**Desktop (please complete the following information):**
 - OS: macOS
 - GAMA version: git

**Additional context**
-

**Comment by AlexisDrogoul** [comment]
This problem is a direct consequence of the use of some JAI-related classes used to solve #2777 . There aren't a lot of things I can do except, maybe, reverting this fix...

**Comment by AlexisDrogoul** [comment]
The fixes to this issue and #2777 appear to be incompatible. As soon as we need to use some ImageIO / JAI extensions to display the image in #2777 (or to compute its metadata: that's probably what is happening when creating a new workspace), it seems to disable the correct functioning of the `GeoTiffReader` used by `GamaGridFiles`... Incompatible versions ? incompatible plugins ? I really don't know...
